src/lightning_classify/transforms/transforms.py

```diff
@@ -0,0 +1,33 @@
+from PIL import Image
+
+import torch
+from torchvision import transforms
+
+
+class ImageTransform:
```

Comment:
hmmm, just make sure you can make a good reasoning later for using the transform as class

Reply:
"The reason is to simplify the number of function transformations. The test and validation transformations are the same, and the required transformation depends on whether the image is for training or not."